doc/direction.md

```diff
@@ -0,0 +1,63 @@
+# Minetest Direction Document
+
+## 1. Background
+
+To give context to this document, you should consider reading the following
```

Comment:
The documents linked in this 'Background' section are not really 'context for Minetest's direction', they are just as much part of Minetest's direction as our recently discussed priorities.
The wording 'To give context to this document' seems to be undermining his authority and the official roadmaps.

What is linked here in the 'Background' section should be described as other parts of the official roadmap.